postgrator.json

```diff
@@ -2,7 +2,7 @@
   "migrationPattern": "migrations/*",
   "driver": "pg",
   "host": "127.0.0.1",
-  "port": 5432,
+  "port": 5433,
```

Comment:
thought: maybe out of context for the workshop, but I noticed these deprecation warnings from Postgrator:

```bash
postgrator deprecated Config option "port". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "host". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "username". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "password". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "migrationDirectory". use "migrationPattern" instead using glob match. e.g. path.join(__dirname, '/migrations/*') node_modules/postgrator-cli/postgrator-cli.js:221:26
```

Reply:
yeah good catch, postgrator's API has changed, let's reflect those changes here as well

Reply:
We are using `postgrator-cli` latest version and not `postgrator` directly. 
`postgrator-cli` is still using postgrator 4.0. I don't get any warnings with the current version.
@simoneb Fastify workshop also uses the same version of `postgrator-cli`. Should I leave it for now till they upgrade or do you want me to use postgrator 5.0 directly?

Reply:
we're good then 👌